wazp/callbacks/metadata.py

```diff
@@ -346,9 +346,21 @@ def create_metadata_table_and_buttons(
                 ),
             )
 
+            # check for missing metadata files
+            check_missing_files_tooltip = dbc.Tooltip(
+                "Check which videos in the "
+                "video directory have a missing YAML file "
+                "and add a row for each of them. Note that "
+                "this won't generate any YAML files.",
```

Comment:
How about
```suggestion
                "Check which videos in the "
                "video directory are metadata "
                "and add a row for each of them. Note "
                "this won't save the metadata.",
```
...? Better?